conf/locale/fr/LC_MESSAGES/packetfence.po

```diff
@@ -177,6 +177,9 @@ msgstr "Inscription"
 msgid "guest network disclaimer"
 msgstr "Ce réseau public est fournis exclusivement pour la convenence des invités et des visiteurs.  L'accès n'implique aucune garantie de fiabilité, de vitesse ou de confidentialité.  Ce réseau est strictement fournis pour donner un accés a Internet.  Son utilisation implique l'acceptation de tous les termes et conditions contenus dans notre <b>Politique d'accès réseau</b>."
 
+msgid "tls enrolment network"
+msgstr "Réseau destiné à la génération d'un certificat eap-tls. Veuillez remplir les informations suivantes pour etre fourni par votre certificat et votre profile vous permettant de vous connecter au réseau secure."
```

Comment:
Same as above but the french version sounds like a 1 to 1 translation of bad english....